maths/allocation_number.py

```diff
@@ -15,9 +15,9 @@ def allocation_num(number_of_bytes: int, partitions: int) -> list[str]:
     :param partitions: the number of partition need to be allocated.
     :return: list of bytes to be assigned to each worker thread
     >>> allocation_num(16647, 4)
-    ['0-4161', '4162-8322', '8323-12483', '12484-16647']
+    ['0-4161', '4162-8323', '8324-12485', '12486-16647']
     >>> allocation_num(50000, 5)
-    ['0-10000', '10001-20000', '20001-30000', '30001-40000', '40001-50000']
+    ['0-9999', '10000-19999', '20000-29999', '30000-39999', '40000-50000']
```

Comment:
49999 or the function is still allocating one byte too many.

Reply:
Some number can't be divisible.This is normal.